src/interfaces/ILBPairExt.sol

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.20;
+import {ILBFlashLoanCallback} from "./ILBFlashLoanCallback.sol";
+
+interface ILBPairExt {
+    function swap(bool swapForY, address to) external returns (bytes32 amountsOut);
```

Comment:
this function is one of the most used - pls move it back to the LBPair contract.